app/controllers/api/articles_controller.rb

```diff
@@ -11,6 +11,11 @@ def create
     end
   end
 
+  def index
+    articles = Article.all
+    render json: articles
```

Comment:
use a serializer and also you need to return an array wrapped in a keyword, this is not restful

Reply:
we have done that now :)